convert2rhel/actions/system_checks/convert2rhel_latest.py

```diff
@@ -229,10 +267,39 @@ def run(self):
                         diagnosis=(
                             "You are currently running %s and the latest version of convert2rhel is %s.\n"
                             "Only the latest version is supported for conversion."
-                            % (installed_convert2rhel_version, latest_available_version[1])
+                            % (formatted_convert2rhel_version, formatted_available_version)
                         ),
                         remediation="If you want to ignore this check, then set the environment variable 'CONVERT2RHEL_ALLOW_OLDER_VERSION=1' to continue.",
                     )
                     return
 
         logger.info("Latest available convert2rhel version is installed.")
+
+
+def _format_EVR(epoch, version, release):
+    return "%s:%s-%s" % (epoch, version, release)
+
+
+def _extract_convert2rhel_versions(precise_raw_version):
+    temp_raw_output = []
```

Comment:
I would make these variable names relevant to the _extrcat_convert2rhel_versions function rather than relevant to the code that calls them.

```suggestion
def _extract_convert2rhel_versions(raw_version):
    parsed_versions = []
```
Inside of this function, we are taking a string that came straight from repoquery or rpm (thus `raw_version`).
Then we are parsing the string to return a lit of NEVRAs. (Thus, `parsed_versions`.  Other options could be `parsed_nevras`, `versions`, `nevras`)

And of course, change the variable names to match what you choose to do here.

Reply:
I have made a few changes to the names in this function, but now that I am re-reading this comment I will adjust some of the other names to betting fit what this comment is saying!